### PR TITLE
Ori makes list

### PR DESCRIPTION
**Pull request by orikalin**
rewrote the list, added check boxes

### CODE REVIEW

File: src/components/ItemList.jsx [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
-`;
 
-function ItemList({ top, height, list, onRemoveItem, editMode}){
-    return (
-      <Container top={top} height={height}>
-        {list.map((item) => <Item key={item.id} item={item} editMode={editMode} onRemoveItem={onRemoveItem}/>)}
-      </Container>
-    );
-}
+  //var listOfNames = [];
```
Comment (robinmaypanpan) [inline src/components/ItemList.jsx:6]: Can you delete this unused code?

File: src/routes/ViewList.jsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ const Header = styled.header`
   position: sticky;
   top: 0;
   width:100vw;
-
+  z-index: 10;
```
Comment (robinmaypanpan) [inline src/routes/ViewList.jsx:19]: I can't believe this fixed this.

File: src/themes/autumn.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ const autumnColors = {
     dataText: 'white',
     iconColor: 'black',
     highlight: '#F3BC2E',
+
```
Comment (robinmaypanpan) [inline src/themes/autumn.js:9]: Why the extra space?

**Review by robinmaypanpan — APPROVED** [review]
Looks good

File: src/themes/autumn.js (hunk 3 above)
Comment (orikalin) [reply src/themes/autumn.js:9]: oops

**Pull request merged**